homeassistant/components/openhardwaremonitor/sensor.py

```diff
@@ -30,8 +30,8 @@
 CONF_INTERVAL = "interval"
 
 MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=15)
-SCAN_INTERVAL = timedelta(seconds=30)
-RETRY_INTERVAL = timedelta(seconds=30)
+SCAN_INTERVAL = timedelta(seconds=1)
+RETRY_INTERVAL = timedelta(seconds=1)
```

Comment:
The `RETRY_INTERVAL` constant is never used by anything. It's not a constant that is part of the platform interface in contrast to `SCAN_INTERVAL`.